modules/ztp-creating-the-policygentemplate-cr.adoc

```diff
@@ -26,6 +26,23 @@ Depending on the specific requirements of your clusters, you might need more tha
 ====
 
 . Define all the policy namespaces in a YAML file similar to the example `out/argocd/example/policygentemplates/ns.yaml` file.
++
+[IMPORTANT]
```

Comment:
Could this whole admonition be a prerequisite? Having to include a list makes it seem like a candidate, IMO.